### PR TITLE
Key input for external

### PR DESCRIPTION
**Pull request by marcushanses**

### CODE REVIEW

File: external.html [hunk 1]
```diff
@@ -102,7 +102,7 @@
 
 		// Function used in this example to perform 3D. A public api key with 3D configurations is needed to test this example.
 		async function perform3d() {
-			const [status, body] = await post(baseUrl + "/verification?method=GET", "<public-api-key-with-3d-config>", verification)
+			const [status, body] = await post(baseUrl + "/verification?method=GET", document.getElementById("key").value, verification)
```
Comment (robinnedergard) [inline external.html:105]:
```suggestion
			const [status, body] = await post(baseUrl + "/verification?method=GET", document.getElementById("key").value.trim(), verification)
```

File: external.html [hunk 2]
```diff
@@ -57,11 +57,11 @@
 			authorize(authorization)
 		}
 		async function authorize(creatable) {
-			const [status, body] = await post(baseUrl + "/authorization", key, creatable)
+			const [status, body] = await post(baseUrl + "/authorization", document.getElementById("no3dKey").value, creatable)
```
Comment (robinnedergard) [inline external.html:60]:
```suggestion
			const [status, body] = await post(baseUrl + "/authorization", document.getElementById("no3dKey").value.trim(), creatable)
```

Comment (robinnedergard) [reply external.html:60]: To avoid issues with unintentional added spaces when copy pasting into input

File: external.html (hunk 1 above)
Comment (robinnedergard) [reply external.html:105]: To avoid issues with unintentional added spaces when copy pasting into input

**Review by nilssonemma — APPROVED** [review]

**Pull request merged**